src/com/redhat/art/RpmSpecTest.groovy

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/groovy
+
+import com.redhat.art.RpmSpec
+
+class RpmSpecTest extends GroovyTestCase {
+
+    String samples
+
+    void setUp() {
+        samples = [
+            '''# Filler
+Version: 3.0
+Release: 1
+%changelog
+#
+Here is some stuff
```

Comment:
Stuff? This is not final text is it?

Reply:
Actually that's perfectly good final text. This is a mock, not something that will actually commit
 Now I do actually have to *write the tests*...

Reply:
Removed null test file